docs/e2e/appinsights.md

```diff
@@ -49,3 +50,13 @@ requests
 | where id =="e2e-upgrade-asalkeld-1386-phmfxi"
 | render timechart
 ```
+
+
+### Get specific resourcegroup test logs
+
+```
+traces
+| where customDimensions.type == "ginkgo"
+| where customDimensions.resourcegroup == "mjudeikis"
```

Comment:
assuming WIP?

Reply:
example. It is per cluster based. Its one of those where you need to construct your own query based on what you need. Like:
`| where message contains "failed by something" ` style of queries when looking for specific failures. 

WIll replace with "cluster_name" :)